Source/diablo.cpp

```diff
@@ -537,6 +537,7 @@ static void SaveOptions()
 	setIniInt("Game", "Enemy Health Bar", sgOptions.Gameplay.bEnemyHealthBar);
 	setIniInt("Game", "Auto Gold Pickup", sgOptions.Gameplay.bAutoGoldPickup);
 	setIniInt("Game", "Adria Refills Mana", sgOptions.Gameplay.bAdriaRefillsMana);
+	setIniValue("Game", "Auto Equip Items", sgOptions.Gameplay.szAutoEquipItems);

```

Comment:
I really, _really_ dislike the fact we are using strings here. 

I think it is a bad idea to do this from an architecture standpoint, even if it makes the ini simpler. I think without proper, decent support for enum flags (or bitsets) to be mapped directly, this is not worth the trouble: it will simplify the ini, but make the code vastly worse.